core/indexer/processTransactions.go

```diff
@@ -248,6 +251,33 @@ func (tdp *txDatabaseProcessor) groupNormalTxsAndRewards(
 	return transactions, rewardsTxs
 }
 
+func (tdp *txDatabaseProcessor) setTransactionSearchOrder(transactions map[string]*Transaction, shardId uint32) map[string]*Transaction {
+	currentOrder := 0
+	shardIdentifier := tdp.createShardIdentifier(shardId)
+
+	for _, tx := range transactions {
+		stringOrder := fmt.Sprintf("%d%d", shardIdentifier, currentOrder)
```

Comment:
The 10th tx of metachain will have the order 110 which will be higher than the 9th of the shard 2 whcih will have the order 49. Does it matter for the logic that you needed from this method?

Reply:
That's right but we don't care about that. We already can have a tie breaker and sort by shard id. We only need to have a rank between items in the same shard and the same indexed bulk

Reply:
ok